src/gluonts/model/predictor.py

```diff
@@ -134,6 +155,79 @@ def from_inputs(cls, train_iter, **params):
         params = {**auto_params, **params}
         return cls.from_hyperparameters(**params)
 
+    def get_backtest_input(
```

Comment:
I don't think this should be a method. `self` isn't used at all.

Reply:
True. If we make this a function, it might also make sense to move this outside of `predictor.py` but I'm not sure where else to put it...